layouts/_default/events.html

```diff
@@ -1,113 +1,279 @@
 {{ define "body-class" }}
-  text-[16px] bg-beige font-serif leading-normal text-g-9
+  text-[16px] bg-white font-serif leading-normal text-g-9
```

Comment:
I would like it if we could reduce the number of body-class variants. Ideally the only thing in here would be some smooth-scroll/scroll margin stuff that is page specific. Don't need to fix this now, just noting for the future.

Reply:
I've cleaned these up. Most are now only declaring font-serif or font-sans. I think we can probably move toward a better way to select fonts deeper in the pages